pkg/registry/apis/secret/accesscontrol.go

```diff
@@ -0,0 +1,225 @@
+package secret
+
+import (
+	"context"
+
+	claims "github.com/grafana/authlib/types"
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	"github.com/grafana/grafana/pkg/apimachinery/utils"
+	secretv0alpha1 "github.com/grafana/grafana/pkg/apis/secret/v0alpha1"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"k8s.io/apiserver/pkg/authorization/authorizer"
+)
+
+const (
+	// SecureValues
+	ActionSecretsManagerSecureValuesWrite    = "secrets-manager.securevalues:write"    // CREATE + UPDATE.
+	ActionSecretsManagerSecureValuesDescribe = "secrets-manager.securevalues:describe" // GET.
+	ActionSecretsManagerSecureValuesList     = "secrets-manager.securevalues:list"     // LIST.
```

Comment:
For RBAC actions we should merge these two into one permissions and use `read` to follow what we have for other permissions. We don't have a `list` action because list is filtered out based on what you can read

Reply:
> We don't have a list action because list is filtered out based on what you can read
  
This was one of my questions. Each app has to implement the filtering for list individually right?